templates/lb-fargate-service/cf.yml

```diff
@@ -421,3 +431,21 @@ Resources:
       Handle: !If [HTTPLoadBalancer, !Ref HTTPWaitHandle, !Ref HTTPSWaitHandle]
       Timeout: "1"
       Count: 0
+
+  AddonsStack:
+    Type: AWS::CloudFormation::Stack
+    Condition: HasAddons
+    Properties:
+      Parameters:
+        Project: !Ref ProjectName
+        Env: !Ref EnvName
+        App: !Ref AppName
+      Tags:
```

Comment:
@kohidave Do you know if this is needed so that the resources in the nested stack also gets tagged? or can I remove them since we tag the parent stack?

Reply:
The parent stack will pass it down to the child stack, I believe.

Reply:
I'll test it out and report back

Reply:
> The parent stack will pass it down to the child stack, I believe.

You're right (Against all odds 😯) ! I removed the `Tags` section, and the tags from the parent stacked were propagated.